website/src/pages/api/admin/status.ts

```diff
@@ -1,17 +1,17 @@
-import { getToken } from "next-auth/jwt";
 import { withRole } from "src/lib/auth";
-import { oasstApiClient } from "src/lib/oasst_api_client";
-import { getBackendUserCore } from "src/lib/users";
+import { createApiClientFromUser } from "src/lib/oasst_client_factory";
 
 /**
  * Returns tasks availability, stats, and tree manager stats.
  */
 const handler = withRole("admin", async (req, res) => {
+  // NOTE: why are we using a dummy user here?
```

Comment:
This is to fetch global stats that apply to all users.  By fetching stats for the current admin user, the server would report stats that take into account what the admin user has done.  This is a hacky solution to simulate a global stats query.